specification/vmwarevirtustream/resource-manager/Microsoft.VMwareVirtustream/preview/2016-09-01-preview/vmwarevirtustream.json

```diff
@@ -0,0 +1,1199 @@
+        "nsxManager": {
+          "type": "string"
+        },
+        "vcsa": {
+          "type": "string"
+        }
+      }
+    },
+    "AddIdentitySourceRequest": {
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "alias": {
+          "type": "string"
+        },
+        "domain": {
+          "type": "string"
+        },
+        "baseUserDN": {
+          "type": "string"
+        },
+        "baseGroupDN": {
+          "type": "string"
+        },
+        "primaryServer": {
+          "type": "string"
+        },
+        "secondaryServer": {
+          "type": "string"
+        },
+        "useSsl": {
+          "type": "boolean"
+        },
+        "username": {
+          "type": "string"
+        },
+        "credential": {
+          "type": "string"
+        }
+      }
+    },
+    "DeleteIdentitySourceRequest": {
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "alias": {
+          "type": "string"
+        },
+        "domain": {
+          "type": "string"
+        }
+      }
+    },
+    "IdentitySource": {
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "alias": {
+          "type": "string"
+        },
+        "domain": {
+          "type": "string"
+        },
+        "baseUserDN": {
+          "type": "string"
+        },
+        "baseGroupDN": {
+          "type": "string"
+        },
+        "primaryServer": {
+          "type": "string"
+        },
+        "secondaryServer": {
+          "type": "string"
+        },
+        "useSsl": {
+          "type": "boolean"
+        }
+      }
+    },
+    "PrivateCloud": {
+      "properties": {
+        "provisioningState": {
+          "type": "string",
+          "enum": [
+            "Succeeded",
+            "Failed",
+            "Cancelled",
+            "Pending",
+            "Building",
+            "Updating"
+          ]
+        },
+        "circuit": {
+          "$ref": "#/definitions/Circuit"
+        },
+        "cluster": {
+          "$ref": "#/definitions/Cluster"
+        },
+        "clusters": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "endpoints": {
+          "$ref": "#/definitions/Endpoints"
+        },
+        "internetEnabled": {
+          "type": "boolean"
+        },
+        "identitySources": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IdentitySource"
+          }
+        },
+        "vpc": {
+          "$ref": "#/definitions/Vpc"
+        }
+      }
+    },
+    "Cluster": {
+      "properties": {
+        "clusterId": {
+          "type": "integer"
+        },
+        "provisioningState": {
+          "type": "string",
+          "enum": [
+            "Succeeded",
+            "Failed",
+            "Cancelled",
+            "Updating"
+          ]
+        },
+        "clusterSize": {
```

Comment:
can this be an enum? it will have fixed set of allowed values, correct?

Reply:
I set `"minimum": 3` & `"maximum": 16` in next API version. Does that work?